modules/transcription-service-amberscript/src/main/java/org/opencastproject/transcription/amberscript/AmberscriptTranscriptionService.java

```diff
@@ -362,20 +358,8 @@ protected String process(Job job) throws Exception {
   }
 
   void createRecognitionsJob(String mpId, Track track, String languageCode, String jobtype) throws TranscriptionServiceException, IOException {
-    CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
-    credentialsProvider.setCredentials(AuthScope.ANY,
-      new UsernamePasswordCredentials(clientKey, ""));
-
```

Comment:
Was this credential provider unnecessary?

Reply:
Yep, see [documentation](https://github.com/amberscript/api). Only api key as query argument is needed.